generator/internal/service_code_generator.h

```diff
@@ -171,9 +171,8 @@ class ServiceCodeGenerator : public GeneratorInterface {
    * the number of method signatures to emit may be fewer than the number in
```

Comment:
Perhaps this comment needs revisiting now?

Reply:
Fixed.